index.js

```diff
@@ -163,8 +186,9 @@ const sortScripts = onObject((scripts, packageJson) => {
       ),
     [],
   )
+  const toReturn = sortObjectKeys(scripts, order)
 
-  return sortObjectKeys(scripts, order)
+  return toReturn
```

Comment:
Since we're only using `toReturn` as a return value, I think we can just keep it as it was?

```suggestion
  return sortObjectKeys(scripts, order)
```